how-to-efficiently-stress-test-pod-memory.md

```diff
@@ -241,6 +241,11 @@ That's all for the tour. When you inject StressChaos into your Pods, we have two
 
 In the future, we'll create a more detailed StressChaos document.
 
+<div class="trackable-btns">
+  <a href="/download" onclick="trackViews('How to Efficiently Stress Test Pod Memory', 'download-tidb-btn-middle')"><button>Download TiDB</button></a>
```

Comment:
```suggestion
<div class="trackable-btns">
```
This blog is not related to TiDB; putting 'download TiDB' here is inappropriate.